lib/manager/pipenv/index.js

```diff
@@ -0,0 +1,10 @@
+const { extractDependencies } = require('./extract');
+const { updateDependency } = require('./update');
+
+const language = 'python';
+
+module.exports = {
+  extractDependencies,
+  language,
+  updateDependency,
+};
```

Comment:
We need to add getArtifacts() here for updating Pipfile.lock

Reply:
I just ran across this comment again earlier today. I decided to follow your advice here. I ended up following the pattern I found in the golang manager code, because it uses getArtifacts too.

I've pushed my changes up here. They don't have any tests yet, but I wanted to get them up here so you could give them a once over to see if it makes sense.

It assumes the existence of a renovate/python image that I assume doesn't exist yet. I can probably help get that set up, though.

Reply:
I plan to make it `renovate/pipenv` image